lib/src/components/buttons/details_button.dart

```diff
@@ -1,24 +1,44 @@
 import 'package:flutter/material.dart';
 import 'package:help_ukraine_widget/src/components/chevron_down.dart';
-import 'package:help_ukraine_widget/src/theme/font_config.dart';
 
-/// It's a button that changes color when you hover over it
+/// It's a button that changes color when you hover over it.
 class DetailsButton extends StatelessWidget {
   /// It's a function that calls on button tap.
   final VoidCallback onTap;
 
-  /// It's a color that changes appearance of text and icon
+  /// It's a color that changes appearance of text and icon.
   final Color color;
 
   /// It's a variable that stores the title of the button.
   final String title;
 
-  ///Constructor
+  /// how much free space should be between title and chevron.
+  final double spacingToChevron;
+
+  /// how much free space should be after chevron.
+  final double spacingAfterChevron;
+
+  /// Child widget
+  final Widget child;
```

Comment:
When looking at it -- not sure we actually need this Widget. It's just a [TextButton](https://api.flutter.dev/flutter/material/TextButton-class.html) with some predefined children.

Not sure we need to keep them predefined -- it's only two occurrences, and even those differ slightly.
For all other Widget variations we would have to implement other widgets anyway.

Reply:
Let's remove `DetailsButton` completely, and use `TextButton` for variations 1 and 2 instead.